### DISCUSSION TITLE
Incorrect links for tutorials/misconfiguration/terraform/

### DISCUSSION THREAD

**Discussion by kderck** [aquasecurity/trivy#5660] [Documentation]
### Description

<img width="677" alt="Screenshot 2023-11-26 at 18 29 31" src="https://github.com/aquasecurity/trivy/assets/98526301/0dd7fe9d-99ec-4d9e-90f0-27e44a632072">

Links to "https://aquasecurity.github.io/trivy/latest/docs/misconfiguration/scanning/" should link to "https://aquasecurity.github.io/trivy/latest/docs/scanner/misconfiguration/"

<img width="715" alt="Screenshot 2023-11-26 at 18 30 53" src="https://github.com/aquasecurity/trivy/assets/98526301/17260682-9e33-4790-86b0-0eb24e1f1cbd">

Links to "https://aquasecurity.github.io/trivy/latest/docs/misconfiguration/options/values/" should link to "https://aquasecurity.github.io/trivy/latest/docs/coverage/iac/terraform/#value-overrides"

<img width="692" alt="Screenshot 2023-11-26 at 18 35 10" src="https://github.com/aquasecurity/trivy/assets/98526301/b7cee761-3c40-443f-bb15-a7d2cbc7b5ca">

Links to "https://aquasecurity.github.io/trivy/latest/docs/misconfiguration/custom/" should link to "https://aquasecurity.github.io/trivy/latest/docs/scanner/misconfiguration/custom/"

### Link

This is the page with broken links: https://aquasecurity.github.io/trivy/v0.47/tutorials/misconfiguration/terraform/

### Suggestions

Links should be updated.

**Comment by kderck** [comment]
Opened a Pull Request: [#5561](https://github.com/aquasecurity/trivy/pull/5661)

**Reply by simar7** [reply]
Thanks.
